var/spack/repos/builtin/packages/crtm/package.py

```diff
@@ -51,6 +51,9 @@ class Crtm(CMakePackage):
 
     license("CC0-1.0")
 
+    version(
```

Comment:
@BenjaminTJohnson For previous versions of CRTM (the `_emc` flavor, we had the following dependencies:
```
    depends_on("crtm-fix@2.3.0_emc", when="@2.3.0 +fix")
    depends_on("crtm-fix@2.4.0_emc", when="@=2.4.0 +fix")
    depends_on("crtm-fix@2.4.0.1_emc", when="@2.4.0.1 +fix")
```
The variant `fix` is still defined for `crtm@v3.1.1-build1`, but no fix files are associated with this version. Is this variant no longer needed (i.e. no fix files needed), or do we also need to add a new version of crtm-fix and the dependency here?

Thanks.

Reply:
@AlexanderRichert-NOAA FYI

Reply:
@BenjaminTJohnson would be the right person for this question.
If I understood correctly, we are aiming to avoid agency specific releases. All users are expected to use the same fix folder with crtm model releases. I could be wrong for this specific case, so let's wait for Ben (currently at AMS).

Reply:
@BenjaminTJohnson  would be the right person for this question.
If I understood correctly, we are aiming to avoid agency specific releases. All users are expected to use the same fix folder with crtm model releases. I could be wrong for this specific case, so let's wait for Ben (currently at AMS).

Reply:
Ok, thanks. If that new fixed-file tarball is available from somewhere, then I can configure the variant correctly here (we need to also fix the crtm package in spack develop in this case).

Reply:
@BenjaminTJohnson Is that tarball with the new fixed files available from somewhere for 3.1.1-build1 (to pull down separately, aka EMC method), or does that get pulled down automatically during the build (aka JCSDA method)? Thanks!

Reply:
Any update? We are still waiting for confirmation about the correct fix files to use, whether they are an external package as for previous EMC versions, whether they are downloaded automatically as part of the CRTM build (not good), etc.

@AlexanderRichert-NOAA @BenjaminTJohnson